src/mono/nuget/Microsoft.NET.Sdk.Mono.Toolchain.Manifest/Microsoft.NET.Sdk.Mono.Toolchain.Manifest.pkgproj

```diff
@@ -0,0 +1,31 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <PropertyGroup>
+    <PackageDescription>The mono toolchain is a set of tasks and executables to be used by other dotnet sdks</PackageDescription>
```

Comment:
Shouldn't this be more like "Internal toolchain package not meant for direct consumption. Please do not reference directly." ?